README.md

```diff
@@ -1,2 +1,55 @@
 # Conduit Connector Firebolt
 
+## General
+
+The Firebolt connector is one of Conduit plugins. It provides both, a source and a destination Firebolt connector.
+
+In order to work with Firebolt the connector needs access to an [engine](https://docs.firebolt.io/working-with-engines/).
+Engines are computed clusters that run database workloads.
+
+If the engine you specified in the connector configuration is not running, the connector will start it for you.
+And it will periodically check the engine status until it starts.
+The process of starting the engine may take some time, the connector at this moment will not be able to write or read data.
```

Comment:
How are failures handled?

Reply:
We check status of database engine each period time using api call, and if it is return error we return this error, if it takes more than 10 minutes(it s mean firebolt stucks. We get this case a few times and only firebolt support resolved this issue ) we return context timeout error.

Reply:
It would be good to add that to the readme.

Reply:
Sure.
Done